app.js

```diff
@@ -104,13 +134,26 @@ function displayFriends() {
         const friendEl = renderFriend(friend);
 
         friendEl.addEventListener('click', () => {
+            if (!mushrooms.length) {
+                message = `not enough mushrooms`;
+            } else if (friend.satsified === 3) {
+                message = `${friend.name} has their mushrooms, pick someone else!`;
+            } else {
+                mushrooms.pop();
+                friend.satisfied++;
+                message = `${friend.name} is satisfied!`;
+            }
```

Comment:
not sure why this isn't working, this looks correct. i'd use console.logs to try and figure out why ```friend.satisfied``` keeps incrementing even if ```friend.satisfied === 3```